README.md

```diff
@@ -86,11 +108,36 @@ python manage.py resources_import --all kirjastot
 
 
 ### Settings
-- `RESPA_IMAGE_BASE_URL`: Base URL used when building image URLs in email notifications. Example value: `'https://api.hel.fi'`.
 
```

Comment:
Add where the settings are set:
Settings are done either by setting environment variables named after the setting or adding them to `.env`-file in the project root. .env-file syntax is similar to TOML-files (INI-files), ie. KEY=VALUE pairs. Project root is the directory where this README is.

Reply:
You can also set settings in local_settings.py, which allows you to set any variables whatsoever. However, some of the settings documented here are named differently in settings.py, especially authentication variables.

Reply:
Very good, sir!